src/Microsoft.Extensions.DependencyInjection.Ordered/project.json

```diff
@@ -0,0 +1,37 @@
+{
+  "version": "1.1.0-*",
+  "description": "IOrdered implementation for for Microsoft.Extensions.DependencyInjection.",
+  "packOptions": {
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/aspnet/dependencyinjection"
+    },
+    "tags": [
+      "dependencyinjection",
+      "di"
+    ]
+  },
+  "buildOptions": {
+    "warningsAsErrors": true,
+    "keyFile": "../../tools/Key.snk",
+    "nowarn": [
+      "CS1591"
+    ],
+    "xmlDoc": true
+  },
+  "dependencies": {
+    "Microsoft.Extensions.DependencyInjection": "1.1.0-*",
```

Comment:
Why does it need the implementation of the default container? That doesn't seem right.